Tests/StreamChatTests/DummyData/ChannelPayload.swift

```diff
@@ -7,7 +7,7 @@ import Foundation
 
 extension ChannelDetailPayload where ExtraData == NoExtraData {
     /// Returns a dummy channel detail payload with the given cid
-    static func dummy(cid: ChannelId) -> ChannelDetailPayload {
+    static func dummy(cid: ChannelId, teamId: TeamId? = nil) -> ChannelDetailPayload {
```

Comment:
Not sure if this is needed tbh :) Why no just assign `.unique` to the `team` variable like we do it with other things?